### DISCUSSION TITLE
Error in overriding configuration parameters

### DISCUSSION THREAD

**Discussion by arun98475** [bluenviron/mediamtx#1109] [Questions]
While trying to override the configuration parameters following error occurs. I would like to change the listening port without editing .yml file directly

 
![image](https://user-images.githubusercontent.com/65711116/185774615-1abcfd2b-842c-4247-bc0e-f6db82ec688a.png)

**Comment by aler9** [comment]
Hello, in order to set environment variables with the `cmd` Windows terminal, you have to use `set envname=val`

i.e.

```
set RTSP_RTSPADDRESS=127.0.0.1:8555
rtsp-simple-server.exe
```

[a comment by an automated account is not shown]
